apiary.apib

```diff
@@ -4755,6 +4755,170 @@ Update the current timer. Only possible if there is a timer running.
+                + subject (object, nullable)
+                    + type: `company` (enum[string])
+                        + Members
+                            + company
+                            + contact
+                            + deal
+                            + invoice
+                            + creditNote
+                            + generalEmailTemplate
+                            + meeting
+                            + order
+                            + product
+                            + project
+                            + quotationEmailTemplate
+                            + ticket
+                            + templateAttachment
+                            + workOrder
+                    + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
+                + name: `meeting-report.pdf` (string)
+                + mime_type (MimeType)
+                + size (FileSize)
+                + updated_at: `2019-10-15T17:30:58+00:00` (string)
+                + uploaded_by (User, nullable)
+                + folder : `Documents` (string)
+
+### files.info [GET /files.info]
+
+Get details for a single file.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `6fac0bf0-e803-424e-af67-76863a3d7d16` (string, required)
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + id: `a856642a-a476-4b69-8e42-e7d1badb2ce9` (string)
+            + subject (object, nullable)
+                + type: `company` (enum[string])
+                    + Members
+                        + company
+                        + contact
+                        + deal
+                        + invoice
+                        + creditNote
+                        + generalEmailTemplate
+                        + meeting
+                        + order
+                        + product
+                        + project
+                        + quotationEmailTemplate
+                        + ticket
+                        + templateAttachment
+                        + workOrder
+                + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string)
+            + name: `meeting-report.pdf` (string)
+            + mime_type (MimeType)
+            + size (FileSize)
+            + updated_at: `2019-10-15T17:30:58+00:00` (string)
+            + uploaded_by (User, nullable)
+            + folder : `Documents` (string)
+
+### files.download [POST /files.download]
+
+Request the download link for a file.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string, required)
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
+            + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download url
```

Comment:
A more fitting name would be `expires_at` here. `expires` is more like a boolean it feels like.

```suggestion
            + expires_at: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download url
```